### PR TITLE
fix: [CVNG-4043]: UI update for verify step in pipeline

### PR DESCRIPTION
**Pull request by praneshg239**
##### Screenshots:

image

You can use the following comments to re-trigger PR Checks

- Jest: `retrigger jest`
- Prettier: `retrigger prettier`
- Type Check: `retrigger typecheck`
- ESLint: `retrigger eslint`
- Standards: `retrigger standards`
- Build: `retrigger build`
- Title Check: `retrigger titlecheck`
- Coverage: `retrigger coverage`
- Rebase: `trigger rebase`
- Cypress: `trigger cypress`
- Fix Prettier - `fix prettier`

### CODE REVIEW

File: src/modules/85-cv/components/ExecutionVerification/ExecutionVerificationView.tsx [hunk 1]
```diff
@@ -34,8 +34,18 @@ export function ExecutionVerificationView(props: ExecutionVerificationViewProps)
...
-          <DeploymentMetrics step={step} selectedNode={selectedNode} activityId={activityId as unknown as string} />
+          <Layout.Horizontal style={{ height: '100%' }}>
+            <ExecutionVerificationSummary
+              displayAnalysisCount={false}
+              step={step}
+              className={css.executionSummary}
+              onSelectNode={setSelectedNode}
+              isConsoleView
+            />
+            <DeploymentMetrics step={step} selectedNode={selectedNode} activityId={activityId as unknown as string} />
```
Comment (deepesh-ui) [inline src/modules/85-cv/components/ExecutionVerification/ExecutionVerificationView.tsx:47]: why we need  `as unknown as string`  activityId is string

Comment (praneshg239) [reply src/modules/85-cv/components/ExecutionVerification/ExecutionVerificationView.tsx:47]: not sure, it was there already.

File: src/modules/85-cv/components/ExecutionVerification/components/DeploymentMetrics/components/DeploymentAccordionPanel/MetricsAccordionPanelSummary.constants.ts [hunk 2]
```diff
@@ -0,0 +1,8 @@
+export enum RISK_TYPES {
```
Comment (deepesh-ui) [inline src/modules/85-cv/components/ExecutionVerification/components/DeploymentMetrics/components/DeploymentAccordionPanel/MetricsAccordionPanelSummary.constants.ts:1]: we already have  RISK TYPES defined at `src/modules/85-cv/utils/CommonUtils.ts`

File: src/modules/85-cv/components/ExecutionVerification/components/DeploymentMetrics/components/DeploymentAccordionPanel/MetricsAccordionPanelSummary.tsx [hunk 3]
```diff
@@ -0,0 +1,61 @@
...
+    <>
+      <Text margin={{ left: 'small' }}>{metricName}</Text>
+      <Text>{transactionName}</Text>
+      <Text>
+        <Icon name={healthSourceTypeToLogo(healthSourceType)} margin={{ right: 'small' }} size={16} />
+        {connectorName}
+      </Text>
+      <Text
+        font={{ variation: FontVariation.TABLE_HEADERS }}
+        className={cx(metricRisk, {
```
Comment (deepesh-ui) [inline src/modules/85-cv/components/ExecutionVerification/components/DeploymentMetrics/components/DeploymentAccordionPanel/MetricsAccordionPanelSummary.tsx:43]: we already have logic to determine color based on status using `getSecondaryRiskColorValue`

File: src/modules/85-cv/components/ExecutionVerification/components/DeploymentMetrics/components/DeploymentMetricsAnalysisRow/DeploymentMetricsAnalysisRow.module.scss [hunk 4]
```diff
@@ -41,14 +31,60 @@
...
+  }
+}
+
+.metricRisk {
+  color: var(--risk-color) !important;
+  border: 1px solid var(--risk-color);
+  padding: var(--spacing-xsmall) !important;
+  width: fit-content;
+  border-radius: 4px;
+
```
Comment (deepesh-ui) [inline src/modules/85-cv/components/ExecutionVerification/components/DeploymentMetrics/components/DeploymentMetricsAnalysisRow/DeploymentMetricsAnalysisRow.module.scss:71]: adding these styles in css will restrict us from using at all places
Rather place this at common place or Use the existing logic

File: src/modules/85-cv/components/ExecutionVerification/components/DeploymentMetrics/components/DeploymentAccordionPanel/MetricsAccordionPanelSummary.constants.ts (hunk 2 above)
Comment (praneshg239) [reply src/modules/85-cv/components/ExecutionVerification/components/DeploymentMetrics/components/DeploymentAccordionPanel/MetricsAccordionPanelSummary.constants.ts:1]: This I declared, but I am not using anywhere.

File: src/modules/85-cv/components/ExecutionVerification/components/DeploymentMetrics/components/DeploymentAccordionPanel/MetricsAccordionPanelSummary.tsx (hunk 3 above)
Comment (praneshg239) [reply src/modules/85-cv/components/ExecutionVerification/components/DeploymentMetrics/components/DeploymentAccordionPanel/MetricsAccordionPanelSummary.tsx:43]: This was my first option, but that didn't work. So I have separate classes.

File: src/modules/85-cv/components/ExecutionVerification/components/DeploymentMetrics/components/DeploymentAccordionPanel/components/NodesCount.module.scss [hunk 5]
```diff
@@ -0,0 +1,41 @@
...
+  --backgroundColor: var(--grey-400);
+  background: var(--backgroundColor) !important;
+  box-shadow: inset 0px 0.5px 4px rgba(96, 97, 112, 0.32);
+  width: $nodeCountDisplaySize;
+  height: $nodeCountDisplaySize;
+  line-height: $nodeCountDisplaySize !important;
+  text-align: center;
+  border-radius: 2px;
+  font-size: 12px !important;
+
```
Comment (deepesh-ui) [inline src/modules/85-cv/components/ExecutionVerification/components/DeploymentMetrics/components/DeploymentAccordionPanel/components/NodesCount.module.scss:25]: we should reuse style for _HEALTHY, _OBSERVE etc

File: src/modules/85-cv/components/ExecutionVerification/components/DeploymentMetrics/components/DeploymentMetricsAnalysisRow/DeeploymentMetricsChartConfig.ts [hunk 6]
```diff
@@ -51,14 +50,27 @@ export function chartsConfig(series: Highcharts.SeriesLineOptions[], width: numb
...
+        fillOpacity: 0.5
       }
     },
     tooltip: {
       formatter: function tooltipFormatter(this: any): string {
-        const title = this.series?.name ? `<p>${this.series?.name}</p><br/>` : ''
-        return `<section class="serviceGuardTimeSeriesTooltip">${title}<p>Value: ${this.y.toFixed(2)}</p></section>`
+        const { baseData, actualTestData } = series[0]
+
+        // eslint-disable-next-line
```
Comment (deepesh-ui) [inline src/modules/85-cv/components/ExecutionVerification/components/DeploymentMetrics/components/DeploymentMetricsAnalysisRow/DeeploymentMetricsChartConfig.ts:64]: Not advice to use // @ts-ignore until  necessary

File: src/modules/85-cv/components/ExecutionVerification/components/DeploymentMetrics/components/DeploymentMetricsAnalysisRow/DeploymentMetricsAnalysisRow.tsx [hunk 7]
```diff
@@ -25,70 +25,56 @@ export interface DeploymentMetricsAnalysisRowProps {
...
+    })
+  }, [])
+
   useLayoutEffect(() => {
     if (!graphContainerRef?.current) {
       return
     }
     const containerWidth = graphContainerRef.current.getBoundingClientRect().width
-    setGraphWidth(containerWidth / 3)
+    setGraphWidth(containerWidth / 3.1)
```
Comment (deepesh-ui) [inline src/modules/85-cv/components/ExecutionVerification/components/DeploymentMetrics/components/DeploymentMetricsAnalysisRow/DeploymentMetricsAnalysisRow.tsx:58]: move this constant and add a meaningful name as in what this values means

Comment (deepesh-ui) [reply src/modules/85-cv/components/ExecutionVerification/components/DeploymentMetrics/components/DeploymentMetricsAnalysisRow/DeploymentMetricsAnalysisRow.tsx:58]: const why3.1 = 3.1

File: src/modules/85-cv/components/ExecutionVerification/components/DeploymentMetrics/components/DeploymentMetricsAnalysisRow/DeeploymentMetricsChartConfig.ts (hunk 6 above)
Comment (praneshg239) [reply src/modules/85-cv/components/ExecutionVerification/components/DeploymentMetrics/components/DeploymentMetricsAnalysisRow/DeeploymentMetricsChartConfig.ts:64]: this value is coming from library and for which we don't have type. So it was throwing.

File: src/modules/85-cv/components/ExecutionVerification/components/DeploymentMetrics/components/DeploymentMetricsAnalysisRow/DeploymentMetricsAnalysisRow.tsx (hunk 7 above)
Comment (praneshg239) [reply src/modules/85-cv/components/ExecutionVerification/components/DeploymentMetrics/components/DeploymentMetricsAnalysisRow/DeploymentMetricsAnalysisRow.tsx:58]: I did, yet to push

**Comment by praneshg239** [comment]
retrigger typecheck

**Review by deepesh-ui — APPROVED** [review]

**Review by shashidhargg — APPROVED** [review]

**Pull request merged**